DataFormats/ParticleFlowReco/src/classes_def_2.xml

```diff
@@ -210,43 +211,7 @@
 
   <class name="edm::Ref<std::vector<reco::PFCluster>,reco::PFCluster,edm::refhelper::FindUsingAdvance<std::vector<reco::PFCluster>,reco::PFCluster> >"/>
 
-  <class name="pftools::CalibrationResultWrapper" ClassVersion="11">
-   <version ClassVersion="11" checksum="3535169833"/>
-   <version ClassVersion="10" checksum="1290295437"/>
-  </class>
-  <class name="pftools::Calibratable"  ClassVersion="10">
-   <version ClassVersion="10" checksum="594741590"/>
-  </class>
-  <class name="pftools::CalibratableElement"  ClassVersion="10">
-   <version ClassVersion="10" checksum="2434874022"/>
-  </class>
-  <class name="pftools::CandidateWrapper"  ClassVersion="10">
-   <version ClassVersion="10" checksum="683143818"/>
-  </class>
-  <class name="edm::Wrapper<std::vector<pftools::Calibratable> >" />
-  <class name="std::vector<pftools::Calibratable>" />
-  <class name="std::vector<pftools::CalibratableElement>" />
-  <class name="std::vector<pftools::CandidateWrapper>" />
-  <class name="std::vector<pftools::CalibrationResultWrapper>"/>
-  <class name="pftools::CaloWindow"  ClassVersion="10">
-   <version ClassVersion="10" checksum="1050281643"/>
-  </class>
-  <class name="pftools::TestCaloWindow"  ClassVersion="10">
-   <version ClassVersion="10" checksum="2905084716"/>
-  </class>
-  <class name="pftools::CaloRing"  ClassVersion="10">
-   <version ClassVersion="10" checksum="999780310"/>
-  </class>
-  <class name="std::map<unsigned int, pftools::CaloRing>"/>
-  <class name="std::vector<pftools::CaloRing>"/>
-  <class name="pftools::CaloEllipse"  ClassVersion="10">
-   <version ClassVersion="10" checksum="3700172067"/>
-  </class>
-  <class name="pftools::CaloBox"  ClassVersion="10">
-   <version ClassVersion="10" checksum="3682355512"/>
-  </class>
-  <class name="std::pair<unsigned int,pftools::CaloRing>" />  
-  
+
```

Comment:
Hi,

Modified on top of the same file in 10_6_X to include the modification in #28514.
Please let me know if there is anything more needed.

Best,
Rajdeep

Reply:
Dear @rchatter , could you please update the title with a more explicit one ?  What is here is good for the description box, but there is no way we can guess if this PR is for JetMet, MiniAOD, Tracker or ... ECAL when it will appear in the release notes. Mentioning the purpose of this backport in the title would be appreciated (as a suggestion you can mention the title of the original PR) - Thank you very much.

Reply:
Dear @boudoul 

Done.

Rajdeep